recipes/tests.rb

```diff
@@ -290,20 +286,18 @@
   # Test only on MasterServer since on compute nodes we mount an empty /opt/intel drive in kitchen tests that
   # overrides intel binaries.
   if node['cfncluster']['cfn_node_type'] == 'MasterServer'
-    unless node['cfncluster']['os'] == 'centos6'
-      bash 'check intel mpi version' do
-        cwd Chef::Config[:file_cache_path]
-        code <<-INTELMPI
-          set -e
-          # Initialize module
-          # Unset MODULEPATH to force search path reinitialization when loading profile
-          unset MODULEPATH
-          # Must execute this in a bash script because source is a bash built-in function
-          source /etc/profile.d/modules.sh
-          module load intelmpi && mpirun --help | grep 'Version 2019 Update 8'
-        INTELMPI
-        user node['cfncluster']['cfn_cluster_user']
-      end
+    bash 'check intel mpi version' do
+      cwd Chef::Config[:file_cache_path]
+      code <<-INTELMPI
+        set -e
+        # Initialize module
+        # Unset MODULEPATH to force search path reinitialization when loading profile
+        unset MODULEPATH
+        # Must execute this in a bash script because source is a bash built-in function
+        source /etc/profile.d/modules.sh
+        module load intelmpi && mpirun --help | grep 'Version 2019 Update 7'
```

Comment:
This test should be checking for Intel MPI U8 with `grep 'Version 2019 Update 8'`

Reply:
Fixed, good catch, I removed it as part of the centos6 deprecation (and the rebase)